LEGO1/lego/legoomni/src/build/legocarbuildpresenter.cpp

```diff
@@ -114,14 +114,14 @@ inline void LegoCarBuildAnimPresenter::Beta10Inline0x100733d0()
 // FUNCTION: BETA10 0x10070ab1
 void LegoCarBuildAnimPresenter::PutFrame()
 {
-	switch (m_unk0xbc) {
-	case 0:
+	switch (m_shelfState) {
+	case 0:	//Selected piece for moving / recoloring / decal
```

Comment:
If you figured out the meaning of these values it might be worth declaring an enum for it (see other classes for examples)